src/imports.md

```diff
@@ -1,6 +1,6 @@
 #### Imports
 
-To import the exports of other modules, `alan` provides two different syntaxes to select the exports to import. There are also two different syntaxes to identify the module you wish to import from. These two syntaxes are completely interchangeable leading to four different syntactic combinations. This sounds more complicated than it actually is.
```

Comment:
😂

Reply:
agreed

Reply:
This is a section I deleted. Should I bring it back?

Reply:
Leave it be or restore this @depombo ?

Reply:
no, I thought the removed comment was funny. let's leave it as-is